source/gloperate/include/gloperate/primitives/InfiniteGroundPlane.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <glm/glm.hpp>
```

Comment:
Please don't use `glm/glm.hpp` here, as it includes the whole glm library. Instead, use `glm/fwd.hpp` here for forward declarations, and include `glm/vec4.hpp` and `glm/mat4x4.hpp` in the `cpp` file.